core/services/relay/evm/mercury/data_source_test.go

```diff
@@ -177,8 +178,9 @@ func TestMercury_Observe(t *testing.T) {
 				assert.Equal(t, head.Number-1, obs.MaxFinalizedBlockNumber.Val)
 			})
 			t.Run("if current block num errored", func(t *testing.T) {
-				h2 := htmocks.NewHeadTracker(t)
-				h2.On("LatestChain").Return(nil)
+				var nilHead *evmtypes.Head
```

Comment:
This workaround is needed due to generics enforcement of H where we would get panic: interface conversion: interface {} is nil, not *types.Head [recovered]

Reply:
FWIW you should be able to pass `(*evmtypes.Head)(nil)` inline, if that is easier to read :shrug:

Reply:
Thanks for the note Jordan. I think that is a cleaner approach - easier to read and shorter